qa/tasks/vstart_runner.py

```diff
@@ -898,6 +902,10 @@ def delete_all_filesystems(self):
         for fs in self.status().get_filesystems():
             LocalFilesystem(ctx=self._ctx, fscid=fs['id']).destroy()
 
+    def delete_fs(self, lookup_fs):
+        for fs in self.status().get_filesystems():
+            if fs["mdsmap"]["fs_name"] == lookup_fs:
+                LocalFilesystem(ctx=self._ctx, fscid=fs["id"]).destroy()
```

Comment:
you won't need this anymore

Reply:
I had to put it in because of this error:
```
2023-06-19 22:39:17,086.086 INFO:__main__:======================================================================
2023-06-19 22:39:17,086.086 INFO:__main__:ERROR: test_fs_creation_incomplete_args (tasks.cephfs.test_misc.TestNewFSCreation.test_fs_creation_incomplete_args)
2023-06-19 22:39:17,086.086 INFO:__main__:----------------------------------------------------------------------
2023-06-19 22:39:17,086.086 INFO:__main__:Traceback (most recent call last):
2023-06-19 22:39:17,086.086 INFO:__main__:  File "/home/dparmar/CephRepo1/ceph/qa/tasks/cephfs/test_misc.py", line 708, in test_fs_creation_incomplete_args
2023-06-19 22:39:17,086.086 INFO:__main__:    self.fail("Expected EINVAL")
2023-06-19 22:39:17,086.086 INFO:__main__:AssertionError: Expected EINVAL
2023-06-19 22:39:17,086.086 INFO:__main__:
2023-06-19 22:39:17,086.086 INFO:__main__:During handling of the above exception, another exception occurred:
2023-06-19 22:39:17,086.086 INFO:__main__:
2023-06-19 22:39:17,087.087 INFO:__main__:Traceback (most recent call last):
2023-06-19 22:39:17,087.087 INFO:__main__:  File "/home/dparmar/CephRepo1/ceph/qa/tasks/cephfs/test_misc.py", line 710, in test_fs_creation_incomplete_args
2023-06-19 22:39:17,087.087 INFO:__main__:    self.mds_cluster.delete_fs(self.TEST_FS)
2023-06-19 22:39:17,087.087 INFO:__main__:  File "/home/dparmar/CephRepo1/ceph/qa/tasks/cephfs/filesystem.py", line 492, in delete_fs
2023-06-19 22:39:17,087.087 INFO:__main__:    Filesystem(ctx=self._ctx, fscid=fs["id"]).destroy()
2023-06-19 22:39:17,087.087 INFO:__main__:    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-06-19 22:39:17,087.087 INFO:__main__:  File "/home/dparmar/CephRepo1/ceph/qa/tasks/cephfs/filesystem.py", line 524, in __init__
2023-06-19 22:39:17,087.087 INFO:__main__:    super(Filesystem, self).__init__(ctx)
2023-06-19 22:39:17,087.087 INFO:__main__:  File "/home/dparmar/CephRepo1/ceph/qa/tasks/cephfs/filesystem.py", line 293, in __init__
2023-06-19 22:39:17,087.087 INFO:__main__:    super(MDSCluster, self).__init__(ctx)
2023-06-19 22:39:17,087.087 INFO:__main__:  File "/home/dparmar/CephRepo1/ceph/qa/tasks/cephfs/filesystem.py", line 228, in __init__
2023-06-19 22:39:17,087.087 INFO:__main__:    self.mon_manager = ceph_manager.CephManager(self.admin_remote, ctx=ctx, logger=log.getChild('ceph_manager'))
2023-06-19 22:39:17,087.087 INFO:__main__:                                                ^^^^^^^^^^^^^^^^^
2023-06-19 22:39:17,087.087 INFO:__main__:  File "/home/dparmar/CephRepo1/ceph/qa/tasks/cephfs/filesystem.py", line 222, in admin_remote
2023-06-19 22:39:17,087.087 INFO:__main__:    first_mon = misc.get_first_mon(self._ctx, None)
2023-06-19 22:39:17,087.087 INFO:__main__:                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-06-19 22:39:17,087.087 INFO:__main__:  File "/home/dparmar/CephRepo1/venv/lib/python3.11/site-packages/teuthology/misc.py", line 979, in get_first_mon
2023-06-19 22:39:17,088.088 INFO:__main__:    mons = get_mon_names(ctx, cluster)
2023-06-19 22:39:17,088.088 INFO:__main__:           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-06-19 22:39:17,088.088 INFO:__main__:  File "/home/dparmar/CephRepo1/venv/lib/python3.11/site-packages/teuthology/misc.py", line 970, in get_mon_names
2023-06-19 22:39:17,088.088 INFO:__main__:    host_mons = [[role for role in roles if is_mon(role)]
2023-06-19 22:39:17,088.088 INFO:__main__:                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-06-19 22:39:17,088.088 INFO:__main__:  File "/home/dparmar/CephRepo1/venv/lib/python3.11/site-packages/teuthology/misc.py", line 970, in <listcomp>
2023-06-19 22:39:17,088.088 INFO:__main__:    host_mons = [[role for role in roles if is_mon(role)]
2023-06-19 22:39:17,088.088 INFO:__main__:                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-06-19 22:39:17,088.088 INFO:__main__:  File "/home/dparmar/CephRepo1/venv/lib/python3.11/site-packages/teuthology/misc.py", line 970, in <listcomp>
2023-06-19 22:39:17,088.088 INFO:__main__:    host_mons = [[role for role in roles if is_mon(role)]
2023-06-19 22:39:17,088.088 INFO:__main__:                                            ^^^^^^^^^^^^
2023-06-19 22:39:17,088.088 INFO:__main__:  File "/home/dparmar/CephRepo1/venv/lib/python3.11/site-packages/teuthology/misc.py", line 448, in _is_type
2023-06-19 22:39:17,088.088 INFO:__main__:    role_cluster, role_type, _ = split_role(role)
2023-06-19 22:39:17,088.088 INFO:__main__:                                 ^^^^^^^^^^^^^^^^
2023-06-19 22:39:17,088.088 INFO:__main__:  File "/home/dparmar/CephRepo1/venv/lib/python3.11/site-packages/teuthology/misc.py", line 378, in split_role
2023-06-19 22:39:17,088.088 INFO:__main__:    type_, id_ = role.split('.', 1)
2023-06-19 22:39:17,088.088 INFO:__main__:    ^^^^^^^^^^
2023-06-19 22:39:17,088.088 INFO:__main__:ValueError: not enough values to unpack (expected 2, got 1)
```

Reply:
Okay, I get it now - https://github.com/ceph/ceph/pull/51993#discussion_r1234200435